### PR TITLE
Written tests for typemin/typemax

### PR DESCRIPTION
**Pull request by shikhargoswami**

### CODE REVIEW

File: test/int.jl [hunk 1]
```diff
@@ -395,3 +395,22 @@ end
...
+    @test typemin(Int64) == Int64(-9223372036854775808)
+    @test typemin(UInt64) == UInt64(0)
+    @test typemax(Int8) == Int8(127)
+    @test typemax(UInt8) == UInt8(255)
+    @test typemax(Int16) == Int16(32767)
+    @test typemax(UInt16) == UInt16(65535)
+    @test typemax(Int32) == Int32(2147483647)
+    @test typemax(UInt32) == UInt32(4294967295)
+    @test typemax(Int64) == Int64(9223372036854775807)
+    @test typemax(UInt64) == UInt64(0xffffffffffffffff)
```
Comment (simeonschaub) [inline test/int.jl:415]: Mind also adding `(U)Int128` while you're at it?

**Review by simeonschaub — APPROVED** [review]
Thanks for this! Looks like we were already testing these indirectly in the string printing tests, but makes sense to me to add explicit tests.

File: test/int.jl (hunk 1 above)
Comment (shikhargoswami) [reply test/int.jl:415]: Sure, doing it now :)

**Comment by shikhargoswami** [comment]
@simeonschaub Also, wouldn't it be better and fast to put comments of where tests of that module are written because as you said these tests were somewhere written in a different file. I would be glad to document it as much as I can.

**Comment by simeonschaub** [comment]
I was talking about the tests [here](https://github.com/JuliaLang/julia/blob/82d79ce18f88923c14d322b70699da43a72e6b32/test/numbers.jl#L236). Those were probably mainly for testing the printing of integers in different bases though.

**Comment by shikhargoswami** [comment]
Got it. Thanks for sharing.

**Pull request merged**